packages/teraslice/src/lib/workers/helpers/worker-shutdown.ts

```diff
@@ -39,17 +39,13 @@ export function shutdownHandler(
         || 'unknown-assignment';
 
     const clusteringType = get(context, 'sysconfig.teraslice.cluster_manager_type');
-    const isK8s = clusteringType === 'kubernetes' || clusteringType === 'kubernetesV2';
+    const isK8s = clusteringType === 'kubernetesV2';
     // this is native clustering only
     const isProcessRestart = process.env.process_restart;
-    // everything but the k8s execution_controller should not be allowed be allowed to
+    // everything but the k8s execution_controller should not be allowed to
```

Comment:
I think I fixed this comment incorrectly
```suggestion
    // everything but the k8s execution_controller should be allowed to
```

Reply:
Now that I understand this correctly, I removed the `allowNonZeroExitCode` check. It was only in kubernetes V1 where the execution controller had to exit with zero.